artifacts/definitions/SUSE/Linux/Events/UserGroupMembershipUpdates

```diff
@@ -0,0 +1,65 @@
+name: SUSE.Linux.Events.UserGroupMembershipUpdates
+description: |
+  This artifact collects user group changes from auditd events, by
+  monitoring updates made to either /etc/group or /etc/nsswitch.conf.
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
+
+  On SUSE operating systems you can install it using:
+  ```
+  zypper install audit
+  ```
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT_EVENT
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: pathToAuditctl
+    default: /sbin/auditctl
+    description: We depend on auditctl to install the correct process execution rules.
+
+sources:
+  - query: |
+     // Install the auditd rule if possible.
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-d", "task,never"])
+
+     // add rule to monitor changes to /etc/group
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-w",
+          "/etc/group", "-p", "w", "-k", "vrr_etc_group"])
+
+     // add rule to monitor changes to /etc/nsswitch.conf
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-w",
+          "/etc/nsswitch.conf", "-p", "w", "-k", "vrr_etc_nsswitch_conf"])
+
+     LET change_log = SELECT timestamp(string=Timestamp) AS Time, Sequence,
+           atoi(string=Process.PID) AS Pid,
+           atoi(string=Process.PPID) AS Ppid,
+           Process.PPID AS PPID,
+           atoi(string=Summary.Actor.Primary) AS UserId,
+           Result AS State,
+           Process.Title AS CmdLine,
+           Process.Exe AS Exe,
+           Process.CWD AS CWD,
+           hash(path=Process.Exe) AS Hash
```

Comment:
You can remove Pid, Ppid, PPID, Result, Process.Title, Process.Exe and hash(path=Process.Exe) as we are not displaying those fields on UI

Reply:
good point!